doc/QuickStart.md

```diff
@@ -29,23 +32,48 @@ Installing
 
 ### Prerequisites
 
-#### Installing CPAN modules
+#### Requirements *(root)*
+
+	$ dnf -y install perl perl-CPAN git screen crontabs nano
+> or 		$ aptitude install perl perl-CPAN git screen crontabs nano
+> or 		$ apt-get install perl perl-CPAN git screen crontabs nano
+
+	$ adduser -m pbot
+	
+
+#### Installing CPAN modules *(root)*
```

Comment:
Root isn't necessarily required. It is possible to install CPAN modules to your home directory too.